### DISCUSSION TITLE
中文翻译进度一览 & 词汇统一

### DISCUSSION THREAD

**Discussion by mattholy** [microsoft/ML-For-Beginners#201] [Translations]
# 欢迎来到中文翻译小组！
建议翻译的PR是 #189 

为了让所有的翻译更简洁、明了、统一且易于理解，我们希望可以建立一些共识，设置一些基本的准则。

## 如何合作？
在选择你要翻译的内容前，请检查这个这个讨论组下方的留言，根据它来选择还未翻译的内容。当你选定具体要翻译的内容后，请在这里告诉大家你选择要翻译的内容，以避免重复翻译。
在翻译前，请fork这个仓库，并在一个新的branch中进行翻译。**其中用到的频繁出现的课程中的内容，请参考「规范翻译表」**。
你也可以检查其他人已经翻译完成的内容，关于具体要如何检查，请参考「如何检查」。

**此外，请注意，我们不希望你直接采用谷歌翻译或是其它翻译工具的结果，那样会对读者造成很大困扰。你可以采用翻译工具的结果作为第一版翻译，然后再此基础上进行润色。**

## 如何检查？
你可以对其他人已经翻译完成的内容进行检查。你可以检查这些方面：

* Markdown 语法
* 图片或对其他文档的链接
* 错字（如「机器学习」被打成「机器心心」）
* 专有名词的错误翻译（如「Supervised Learning」被翻译为「在管理下学习」）
* 过度翻译或错误翻译
* 不通顺的语句
* **是否仅仅采用了工具翻译**

如果你对于某一个翻译点并不确定，请在这里讨论，不要开新的Issue或Discussion。

## 规范翻译表
有些词汇在整个课程中多次出现。为了整体统一，减少理解成本，在这里我们维护一个规范翻译表。课程中频繁出现的这些词汇请大家统一翻译成一样的词汇。如果你想修改现有的翻译，或添加新的词汇在这里，请在这个讨论中留言。

英语原文|中文翻译
------------ | -------------
Regional topic|本节主题
Lessons|课程安排
Pre-lecture quiz|课前测验
Post-lecture quiz|课后测验
Assignment|作业
Credits|课程作者

## 致谢
* 本规则灵感来自于法语翻译小组 #186 
* 感谢各位的参与和努力❤️！

### 0.Project Document
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ] [README.md](https://github.com/microsoft/ML-For-Beginners/tree/main/translations) 参与翻译：@mrmrmrfinch

### 1. Introduction
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ] [1.0. Topic](https://github.com/microsoft/ML-For-Beginners/blob/main/1-Introduction/translations/) 参与翻译：@feiyun0112 
* [ ]  [1.1. Intro to ML](https://github.com/microsoft/ML-For-Beginners/tree/main/1-Introduction/1-intro-to-ML/translations) 参与翻译：@feiyun0112 
* [ ]  [1.2. History of ML](https://github.com/microsoft/ML-For-Beginners/tree/main/1-Introduction/2-history-of-ML/translations) 参与翻译：@feiyun0112 
* [ ]  [1.3. Fairness](https://github.com/microsoft/ML-For-Beginners/tree/main/1-Introduction/3-fairness/translations) 参与翻译：@feiyun0112 
* [ ]  [1.4. Techniques of ML](https://github.com/microsoft/ML-For-Beginners/tree/main/1-Introduction/4-techniques-of-ML/translations) 参与翻译：@feiyun0112 

### 2. Regression
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ] [2.0. Topic](https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/translations) 参与翻译： @feiyun0112 
* [ ]  [2.1. Tools](https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/1-Tools/translations) 参与翻译： @feiyun0112 
* [ ]  [2.2. Data](https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/2-Data/translations) 参与翻译： @feiyun0112 
* [ ]  [2.3. Linear](https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/3-Linear/translations) 参与翻译： @feiyun0112 
* [ ]  [2.4. Logistic](https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/4-Logistic/translations) 参与翻译： @feiyun0112 

### 3. Web App
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ]  [3.1. Topic]() 参与翻译：@feiyun0112
* [ ]  [3.1. Web App]() 参与翻译：@feiyun0112

### 4. Classification
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ]  [4.0. Topic](https://github.com/microsoft/ML-For-Beginners/blob/main/4-Classification/translations/README.zh-cn.md) 参与翻译：@feiyun0112 @mattholy 
* [ ]  [4.1. Introduction]() 参与翻译：@feiyun0112
* [x]  [4.2. Classifiers 1]() 参与翻译：@feiyun0112 @XiaojianTang 
* [ ]  [4.3. Classifiers 2]() 参与翻译：@feiyun0112
* [ ]  [4.4. Applied]() 参与翻译：@feiyun0112

### 5. Clustering
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ]  [5.1. Visualize]() 参与翻译：@nikan1996
* [ ]  [5.2. K-Means]() 参与翻译：@nikan1996

### 6. NLP
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ]  [6.1. Introduction to NLP]() 参与翻译：@mrmrmrfinch
* [ ]  [6.2. Tasks]() 参与翻译：@mrmrmrfinch
* [ ]  [6.3. Translation Sentiment]() 参与翻译：@mrmrmrfinch
* [ ]  [6.4. Hotel Reviews 1]() 参与翻译：@mrmrmrfinch
* [ ]  [6.5. Hotel Reviews 2]() 参与翻译：@mrmrmrfinch

### 7. Time Series
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ]  [7.1. Introduction]() 参与翻译：@nikan1996
* [ ]  [7.2. ARIMA]() 参与翻译：@nikan1996

### 8. Reinforcement
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ]  [8.1. Q-Learning]() 参与翻译：@swartz-k
* [ ]  [8.2. Gym]() 参与翻译：@swartz-k

### 9. Real World
校对：
* [ ] 完成校对 参与校对：@mattholy

翻译：
* [x]  [9.0. Topic](https://github.com/microsoft/ML-For-Beginners/tree/main/9-Real-World/translations) 参与翻译：@duanyang25
* [ ] [9.1. Applications](https://github.com/microsoft/ML-For-Beginners/tree/main/9-Real-World/1-Applications/translations) 参与翻译：@mattholy

### Quizzes
校对：
* [ ] 完成校对 参与校对：

翻译：
* [ ]  [Quizzes]() 参与翻译：

## 一些建议
初次开始翻译的朋友建议在[我的仓库](https://github.com/mattholy/ML-For-Beginners)提交，以便将大家的翻译成果按照规范统一起来（例如**校对**、统一润色、修改链接url等。我已经将所有需要翻译的md文件，在对应的translation文件夹下面创建了名为[WIP]README.zh-cn.md的拷贝，请直接修改WIP文件，完成翻译之后移除WIP前缀即可。

**Comment by mattholy** [comment]
我先认领第九章好了

**Reply by yongyizang** [reply]
那我认领第六章吧

**Comment by mattholy** [comment]
另外我觉得有些图片里面的信息还是挺多的，在线蹲一个大触改图

**Reply by fribble186** [reply]
我可以把第一章的图片试着改一改

**Reply by yongyizang** [reply]
@jlooper Could we translate the images to Chinese as well? If we could, is it possible for us to get the original "template" of the images?

**Reply by jlooper** [reply]
hi, @mrmrmrfinch - these are done in procreate by myself or @girliemac - there's no template but if you have ideas on how we. could localize them without redrawing, I'd love to see.

**Comment by fribble186** [comment]
我可以参与校对的工作吗，翻译好的章节我都重新看一下，有争议的地方在这个讨论区和大家一起讨论？

**Reply by yongyizang** [reply]
我感觉现在校对的话，最要紧的事情可能还是先把所有人没统一的翻译统一成一个版本...
以及还有一个事儿，我们是用 “” 还是「」

**Reply by fribble186** [reply]
好的，我看好像大部分章节已经有人认领翻译了，不知道他们作业会不会也翻译，还是只翻译readme部分；另外关于符号问题，我站 “” 哈哈哈

**Reply by mattholy** [reply]
校对的话其实已经翻译完的可以开始校对了。一边校对一边列一个词语对照表

**Reply by newforesee** [reply]
一处误输入：[1-intro-to-ML](https://github.com/microsoft/ML-For-Beginners/blob/main/1-Introduction/1-intro-to-ML/translations/README.zh-cn.md) 第24行，url开头多打了一个中括号导致链接不可用
`- **探索 Scikit-learn**. 熟悉 [Scikit-learn]([https://scikit-learn.org/stable/user_guide.html)，我们在这些课程中引用的一组 ML 库。`

**Comment by yongyizang** [comment]
那个... 大家如果用了谷歌翻译或者是什么机器翻译的话，在改成能用的版本的时候一定一定一定要注意语序哈... 我在改 6-1 的 readme，发现里面有很多类似于「进行一个美美的吃」这种语序...

**Reply by fribble186** [reply]
我也想吐槽一下，看之前的 readme 有挺多看得挺吃力的地方的，大段大段的谷歌翻译，我个人觉得如果只是谷歌翻译搬过来没什么大意义的，谷歌浏览器自带翻译功能的，如果只是搬过来，还不如不翻译。同时也不是不能用谷歌翻译，我觉得至少想楼主说的那样，至少自己读过一遍，最基本的语序要正确，不至于我看着之前的readme，我还不如去看英文原版readme。

**Comment by yongyizang** [comment]
Knowledge Check 咋翻？

**Reply by yongyizang** [reply]
「检查你的信息！」听起来就很班主任（而且很机器翻译）（而且很「你改悔罢！」）
「知识检查站」听起来又很像课本上那种用 Office 2003 的艺术字
「你是否知道...?」又很宜家
怎么看都不是很容易翻译的样子

**Reply by mattholy** [reply]
知识清单？

**Reply by yongyizang** [reply]
他那个意思好像是要 在这里检查一下你有没有获得这些知识 没有就回去看 差不多是这种意思

**Reply by mattholy** [reply]
对啊，听起来就像是清单嘛

**Comment by mattholy** [comment]
Credits我觉得应该翻译为 致谢

**Comment by fribble186** [comment]
今天我来翻译第二章的作业吧

**Reply by fribble186** [reply]
在assignment中经常会出现的短语，我做了一下整理，大家能帮我看看这样翻译准确吗
Instructions               => 说明
Rubric                       => 评判标准
Criteria                      => 标准
Exemplary                 => 优秀
Adequate                  => 中规中矩
Needs Improvement => 仍需努力

**Reply by mattholy** [reply]
我看可以

**Comment by jlooper** [comment]
Hi, everyone, just a note that I rely on you for good translations - I can’t verify them really myself, not being fluent, so if you can avoid Machine translations that’s ideal - 🙏 thank you

**Reply by mattholy** [reply]
Or maybe you can just deny those PRs not from this discussion？

**Reply by jlooper** [reply]
Here's what I'll do - I'll send any new reviewers to this discussion to get a review

**Reply by mattholy** [reply]
Good idea, thanks

**Comment by LostClue** [comment]
我可以认领第五章吗

**Reply by mattholy** [reply]
当然是可以的

**Comment by jlooper** [comment]
Hi, could someone review this please? https://github.com/microsoft/ML-For-Beginners/pull/238

**Reply by mattholy** [reply]
working

**Reply by mattholy** [reply]
@feiyun0112 你认领的章节~

**Reply by jlooper** [reply]
Does it look ok?

**Reply by mattholy** [reply]
> Does it look ok?

yes👌

**Comment by jlooper** [comment]
Hi, this one too please? https://github.com/microsoft/ML-For-Beginners/pull/246

**Reply by mattholy** [reply]
Hi, too many errors, suggest changes.

**Reply by XiaojianTang** [reply]
updated，please take a look.

**Comment by XiaojianTang** [comment]
请问下大家的cheatsheet怎么翻呢，“作弊表”太直白，“速查表”又没有翻出cheat的感觉，“小抄”是不是好点，或者就……不翻了？

**Reply by LostClue** [reply]
A cheat sheet (also cheatsheet) or crib sheet is a concise set of notes used for quick reference. 
速查表我觉得还可以

**Comment by jlooper** [comment]
hi, there are 3 new PRs 

https://github.com/microsoft/ML-For-Beginners/pull/256

https://github.com/microsoft/ML-For-Beginners/pull/255

when reviewed and changes are implemented, can you please ping me the 'all good' and I'll merge :) thanks!

**Reply by LostClue** [reply]
@michael1011101  #255 I've reviewed your PR and gave some suggested changes in this PR. Thanks for checking my suggestions!

**Comment by LostClue** [comment]
有人能核对第五章吗？ 我已经做了翻译，可能有一些不合理和疏漏的地方还请一起讨论 https://github.com/microsoft/ML-For-Beginners/pull/256
### 第五章翻译表
| 英语原文              | 中文翻译     |
| --------------------- | ------------ |
| Regional topic        | 本节主题     |
| Lessons               | 课程安排     |
| Pre-lecture quiz      | 课前测验     |
| Post-lecture quiz     | 课后测验     |
| Assignment            | 作业         |
| Credits               | 致谢         |
| Instruction           | 说明         |
| 本节相关              |              |
| Clustering            | 聚类（动词） |
| Cluster               | 聚类（名词） |
| Supervised learning   | 监督学习     |
| Unsupervised learning | 无监督学习   |
| Pattern               | 模式         |
| Inductive             | 归纳的       |
| Transductive          | 转导的       |
| Centroid              | 质心         |
| Clustroid             | 聚类中心     |
| Constrained           | 约束         |

**Reply by mattholy** [reply]
稍等，稍晚些时候校对

**Comment by LostClue** [comment]
第七章时间序列我将在近期翻译~

**Comment by SpokieKid** [comment]
@mattholy 
咱们要不要统一一下进度？issue PR 和discussion各有一个进度更新表，但是都不同步唉。

**Reply by mattholy** [reply]
比较难，PR里面有些并没有在这个discussion里面提出讨论

**Comment by zirnc** [comment]
Hi! #318

**Reply by jlooper** [reply]
merged, thanks!

**Comment by jlooper** [comment]
Could I get a review please? https://github.com/microsoft/ML-For-Beginners/pull/320

**Reply by zirnc** [reply]
He didn't finish the translation :joy:

**Reply by jlooper** [reply]
My bad, I thought I saw the edit…

**Reply by SpokieKid** [reply]
@ChungZH Translation done, please review, thanks.

**Comment by jlooper** [comment]
@mattholy and @SpokieKid i think there are PRs with some requested edits awaiting you 🙏

**Reply by SpokieKid** [reply]
Got it, will get them done today. Sorry for the delay

**Comment by swartz-k** [comment]
有一些强化学习经验，申请翻译第8章

**Reply by swartz-k** [reply]
Hello I have a new PR #360. Thanks.

**Reply by swartz-k** [reply]
做了一些修改，辛苦帮忙review #397 
Made some changes and I reopen it as #397. Thanks for reviewing.

**Comment by jlooper** [comment]
hi everyone, could you take a look at the PRs and make any changes so that I can merge? thanks so much!
